libse/Dictionaries/OcrFixReplaceList.cs

```diff
@@ -113,14 +113,19 @@ public static OcrFixReplaceList FromLanguageId(string languageId)
                 XmlNode node = doc.DocumentElement.SelectSingleNode(name);
                 if (node != null)
                 {
+                    const string ReplaceWith = "replaceWith";
+                    const string Find = "find";
                     foreach (XmlNode item in node.ChildNodes)
                     {
-                        if (item.Attributes != null && item.Attributes["replaceWith"] != null && item.Attributes["find"] != null)
+                        if (item.Attributes != null && item.Attributes[ReplaceWith] != null && item.Attributes[Find] != null)
                         {
-                            string to = item.Attributes["replaceWith"].InnerText;
-                            string from = item.Attributes["find"].InnerText;
-                            if (!list.ContainsKey(from))
-                                list.Add(from, to);
+                            string to = item.Attributes[ReplaceWith].InnerText;
+                            string from = item.Attributes[Find].InnerText;
```

Comment:
Is this OK?
According to [MSDN](https://msdn.microsoft.com/en-us/library/system.xml.xmlattribute.innertext%28v=vs.110%29.aspx) the `XmlAttribute.InnerText` property would be **set** only.
Perhaps, better use the [`XmlAttribute.Value` property](https://msdn.microsoft.com/en-us/library/system.xml.xmlattribute.value%28v=vs.110%29.aspx) ?